node/resolvers/catalog/category.ts

```diff
@@ -8,7 +8,7 @@ const lastSegment = compose<string, string[], string>(
 )
 
 function cleanUrl(url: string) {
-  return url.replace(/https:\/\/[A-z0-9]+\.vtexcommercestable\.com\.br/, '')
+  return url.replace(/(https|http):\/\/[A-z0-9]+\.vtexcommercestable\.com\.br/, '')
```

Comment:
Is it possible that the URL comes in this format: http://www.qualquercoisa.vtexcommercestable.com.br ?
If the answer is yes, this regex does not cover it.